stoservice/src/main/java/com/tutrit/stoservice/provider/OrderProvider.java

```diff
@@ -0,0 +1,27 @@
+package com.tutrit.stoservice.provider;
+
+import com.tutrit.stoservice.bean.Order;
+import com.tutrit.stoservice.repository.OrderRepository;
+
+public class OrderProvider {
+    private OrderProvider() {
+    }
+
+    public static Order getInstanceOrder(Order order, int i) {
+        order.setId("id" + i);
+        order.setUser("UserName");
+        order.setCar("UserCar");
+        order.setOrderStatus("Open");
+        return order;
+    }
+
+    static OrderRepository createTenInstanceOrders() {
```

Comment:
I think that would be covered on the scope of another issue, so could be deleted for now